data/json/regional_map_settings.json

```diff
@@ -58,7 +58,8 @@
         "f_dandelion": 8,
         "f_boulder_small": 1,
         "f_boulder_medium": 0.6666,
-        "f_boulder_large": 0.3333
+        "f_boulder_large": 0.3333,
+        "f_shrub_bee_hive": 0.4166
```

Comment:
I don't think gameplay-significant beehives are more common than large rocks, either. This should probably be around 0.1 or lower.